pkg/oam/labels.go

```diff
@@ -233,6 +233,9 @@ const (
 
 	// AnnotationAddonDefinitionBondCompKey indicates the definition in addon bond component.
 	AnnotationAddonDefinitionBondCompKey = "addon.oam.dev/bind-component"
+
+	// AnnotationTraitStageType is used to specify stage type
+	AnnotationTraitStageType = "trait.oam.dev/stage-type"
```

Comment:
```suggestion
	AnnotationTraitStageType = "trait.oam.dev/stage"
```

Reply:
This is a good suggestion, I will modify it.